src/main/docker/jre/update-java-security-file.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+# Copyright 2024 Open Text.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Find the java.security file
+java_security_file=$(find /etc/java -name "java.security" | grep -v "jre" | sort -r | head -n 1)
+
+if [ -z "$java_security_file" ]; then
+    echo "Error: java.security file not found."
+    exit 1
+fi
+
+echo "Found java.security file at: $java_security_file"
+
+# Make a backup of the original file
+cp "$java_security_file" "${java_security_file}.bak"
+
+# Fix for SecureRandom.getInstanceStrong() not available on FIPS enabled system
+# See: https://issues.redhat.com/browse/RHEL-3478
+if sed -i 's/^securerandom\.strongAlgorithms=.*/securerandom.strongAlgorithms=PKCS11:SunPKCS11-NSS-FIPS/' "$java_security_file"; then
```

Comment:
@raghavendran-gopalakrishnan I notice this is similar to a change you were making in:

https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=946056

I am just updating the value to:

`PKCS11:SunPKCS11-NSS-FIPS` (as suggested here https://issues.redhat.com/browse/RHEL-3478), 

I notice you updated it to include the previous values as well. In [this ](https://github.com/CAFapi/opensuse-base-images/pull/12#issuecomment-2290798473) comment you mention that was required for Jenkins to pass? Do you remember which particular Jenkins build was failing?

Reply:
I was under the impression that retaining the previous values was required for non-FIPS systems like Jenkins. Looking at your successful builds, seems like that's not the case.

Reply:
Thanks, I tried a couple of more test builds on both jenkins and sou-jenkins and they passed:

https://gitlab.otxlab.net/cybersecurity-enterprise/cddri/app/darwin-worker-elastic/-/merge_requests/391

https://gitlab.otxlab.net/cybersecurity-enterprise/cddri/app/darwin-job-service/-/merge_requests/307

also tried deploying those to K8s and all looked fine.